tools/wptrunner/wptrunner/browsers/base.py

```diff
@@ -275,3 +280,123 @@ def __call__(self, line):
         self.logger.process_output(self.pid,
                                    line.decode("utf8", "replace"),
                                    command=" ".join(self.command) if self.command else "")
+
+
+class WebDriverBrowser(Browser):
+    __metaclass__ = ABCMeta
+
+    default_base_path = "/"
+
+    def __init__(self, logger, binary, webdriver_binary, webdriver_args=None,
+                 host="127.0.0.1", port=None, base_path="", env=None, **kwargs):
```

Comment:
Shouldn't it be `base_path=None` instead?

Reply:
I went with `base_path="/"` since that's the actual default value, and we aren't using the "class sets the default" behaviour anywhere (an alternative option would have been "we don't use this at all, time to remove it").